languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -36090,19 +36090,146 @@ though, and has since been slightly modified)-->
       <example correction=","><marker>dans ce cas</marker> il est certain</example>
       <example><marker>dans ce cas</marker>, il est certain</example>
     </rule>
-    <rule default="off" id="ET_CE" name="et ce (sans virgule après)">
-      <pattern>
-        <marker>
-          <token>et</token>
-          <token>ce</token>
-        </marker>
-        <token>
-          <exception regexp="yes">[,.;!:\-?n]</exception></token>
-      </pattern>
-      <message>« Et ce » (sans virgule après) peut être une faute de typographie. Employez une virgule (<suggestion>,</suggestion>) au besoin.</message>
-      <example correction=","><marker>et ce</marker> blah blah</example>
-      <example><marker>et ce</marker>,</example>
-    </rule>
+      <rulegroup default="off" id="VIRGULES_ET_CE" name="virgules et ce">
```

Comment:
temp_off — otherwise it won't get checked at all

And please keep things in the usual order ;)

`<rulegroup id="VIRGULES_ET_CE" name="virgules et ce" default="temp_off" >`

Reply:
I change the rule as required.